src/ActionCreators/AuthActionCreators.js

```diff
@@ -0,0 +1,16 @@
+// Action type
```

Comment:
Just a suggestion, its a common convention to have all actionTypes defined in one file. In the end, all actionTypes for the whole project will have a single point of reference.